community/faas/docs/templates/network.md

```diff
@@ -0,0 +1,40 @@
+# Networks and subnets
```

Comment:
Network template(s) should go into a nested directory - templates/networks.

This file should be the README in that directory.

Reply:
Can directory structure be addressed later on @morgante ?

Reply:
Yes, please do in a follow-up.